benchmark/CMakeLists.txt

```diff
@@ -103,8 +107,9 @@ endif()
 if(LINUX)
     set(LIBS_OPTIONAL ${LIBS_OPTIONAL} m)
 endif()
-if(UMF_BUILD_GPU_TESTS)
+if(UMF_BUILD_GPU_TESTS AND UMF_BUILD_LEVEL_ZERO_PROVIDER)
     set(LIBS_OPTIONAL ${LIBS_OPTIONAL} ze_loader)
+    # TODO add CUDA
```

Comment:
TODO add CUDA

Reply:
I don't want to add CUDA to ubench in this PR